src/components/Nav/GlobalMTLSStatus.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { Component } from '../../store/Store';
+import { Icon } from 'patternfly-react';
+import { style } from 'typestyle';
+import { PfColors } from '../Pf/PfColors';
+
+type Props = {
+  status: { [key: string]: string };
+  components: Component[];
+  warningMessages: string[];
+};
+
+const iconStyle = style({
+  position: 'absolute',
+  left: 190,
+  top: 23,
+  color: PfColors.Green
+});
```

Comment:
Can this styling be moved to CSS? why is it here?

Reply:
About 1): for this entire Istio-Mesh controlled by the Istio control plane we are talking to. And yes, this will include all projects including future ones, that are addressed by this control plane. Currently this means all projects that use Istio, but in the future with soft-multitenancy, there could be more than one Istio installation and each one could treat that differently.
1b: one also would have a matching global DR, that has Istio-mTLS enabled.
If we want to be precise here, we can show a) no lock for no mTLS b) a broken lock if one of { MeshPolicy, global mTLS-DR } is missing, a lock if all is correct.
2) I agree, that should move to the right side of the mast-head

Reply:
> About 1): for this entire Istio-Mesh controlled by the Istio control plane we are talking to. And yes, this will include all projects including future ones, that are addressed by this control plane. Currently this means all projects that use Istio, but in the future with soft-multitenancy, there could be more than one Istio installation and each one could treat that differently.
> 1b: one also would have a matching global DR, that has Istio-mTLS enabled.
> If we want to be precise here, we can show a) no lock for no mTLS b) a broken lock if one of { MeshPolicy, global mTLS-DR } is missing, a lock if all is correct.
> 2) I agree, that should move to the right side of the mast-head

yes, making this more precise would be good.
@lucasponce mentioned that those details should be clarified in the JIRA. I have no objections to that. Butn in order to decide if JIRA is resolved by a PR, the functionality should be detailed and clear.
Otherwise I find it hard to review and it also will be hard for QE to test.